pep-0676.rst

```diff
@@ -121,69 +128,84 @@ The following redirect rules must be created from the `python.org`_ domain:
 =====================
 
-No security implications, and the main `python.org`_ website will no longer
-take raw HTML uploads, closing a potential threat vector.
+The main `python.org`_ website will no longer need to process raw HTML
+uploads, closing a potential threat vector, and the PEP building and
+deployment process will use modern, well maintained code and secure
+automated platforms, reducing the likely potential attack surface. Therefore,
+there is no forseen negative security impact.
+
 
 How to Teach This
 =================
 
 The new canonical URLs will be publicised in the documentation. However, this
 is mainly a backend infrastructure change, and there should be minimal
-end-user impact.
+end-user impact. PEP 1 and PEP 12 will be updated as needed to include and
+document any relevant new roles, directive, syntax and other constructs enabled
+by this change.
+
 
 Reference Implementation
 ========================
 
 The proposed implementation has been merged into the `python/peps`_ repository
-in a series of pull requests [8]_. This automatically renders all PEPs on every
-commit.
+in a series of pull requests [8]_. It currently automatically renders all
+PEPs on every commit.
+
 
 Rejected Ideas
 ==============
 
-It would likely be possible to amend the current (as of November 2021)
-rendering process to include a lot of the quality-of-life improvements and
-issue mitigations mentioned above. However, we do not believe that this would
+It would likely be possible to amend the current docutils-based manual
+rendering process to include many of the quality-of-life improvements and
+bug mitigations mentioned above. However, we do not believe that this would
 solve the distributed tooling issue.
 
 It would be possible to use the output from the proposed rendering system and
-import it into `python.org`_. We would argue however that this would be the
-worst of both worlds, as a great deal of complexity is added, and none is
-removed.
+import it into the existing `python.org`_ domain. We would argue, however,
+that this would be the worst of both worlds, as a great deal of complexity
+is added, while none is removed.
+
 
 Open Issues
 ===========
 
 None.
 
-Acknowledgements
-================
 
-Thanks to Hugo van Kemenade, Pablo Galindo Salgado, and Éric Araujo for support
-since April 2020.
+Acknowledgments
+===============
+
+Thanks to Hugo van Kemenade, Pablo Galindo Salgado and Éric Araujo for
+supporting this effort since April 2020.
+
 
 Footnotes
 =========
 
 .. _documentation: https://pythondotorg.readthedocs.io/pep_generation.html
 .. _docutils: https://docutils.sourceforge.io
 .. _Makefile: https://www.gnu.org/software/make/manual/make.html#Introduction
-.. _peps.python.org: https://peps.python.org/
-.. _peps.python.org/pep-0008/: https://peps.python.org/pep-0008/
 .. _python.org: https://www.python.org
+.. _peps.python.org: https://peps.python.org/
+.. _peps.python.org/pep-0008: https://peps.python.org/pep-0008/
```

Comment:
I was sorting logically (domain -> subdomain -> url) rather than lexigraphically here, but its fine if you want to keep that, of course—I auto-lexigraphically-sorted the references section in my PEP 639, albeit that one was over 100 entries long.